tests/upsert_tests.rs

```diff
@@ -7,7 +7,11 @@ use sea_orm::TryInsertResult;
 use sea_orm::{sea_query::OnConflict, Set};
 
 #[sea_orm_macros::test]
-#[cfg(feature = "sqlx-postgres")]
+#[cfg(any(
+    feature = "sqlx-mysql",
+    feature = "sqlx-sqlite",
+    feature = "sqlx-postgres"
+))]
```

Comment:
Can you add some asserts below, e.g. one line 50 where we select the inserted rows to double check?